README.md

```diff
@@ -67,6 +67,21 @@ versions are not installed).
 Navigate to http://localhost:9966 to view the development environment.
 
 
+### STRTA
+
+This project uses [`scripts-to-rule-them-all`](https://github.com/azavea/architecture/blob/master/doc/arch/adr-0000-scripts-to-rule-them-all.md) to bootstrap, test, and maintain projects consistently across all teams. Below is a quick explanation for the specific usage of each script.
+
+| Script      | Use                                                        |
+| ----------- | ---------------------------------------------------------- |
+| `bootstrap` | Pull down secrets from S3                                  |
+| `dbshell`   | Open a `psql` shell connected to the local dev database    |
+| `infra`     | Execute Terraform subcommands with remote state management |
+| `server`    | Start the frontend and backend services                    |
+| `setup`     | Setup the project development environment                  |
+| `test`      | Run linters and tests                                      |
+| `update`    | Update project, assemble, run migrations                   |
+
```

Comment:
Please let me know if these suffice. I am happy to add more scripts or more descriptions of other scripts that was written before.